### PR TITLE
Feature/asm 10942 make defaults not injectable

### PR DESCRIPTION
**Pull request by dan-akeyless**
This new tag makes the default visible in the documentation but actually ignores it

### CODE REVIEW

File: tag_test.go [hunk 1]
```diff
@@ -9,18 +9,17 @@ import (
...
-		Edit   string `edit:"edit"`
-		Usage  string `cli:"usage" usage:"hello,usage"`
-		Dft    string `cli:"dft" dft:"hello,dft"`
-		Name   string `cli:"name" name:"hello-name"`
-		Prompt string `cli:"prompt" prompt:"hello,prompt"`
-		Parser string `cli:"parser" parser:"json"`
-		Sep    string `cli:"sep" sep:":"`
-		Multi  string `cli:"multi" 
-			usage:"multi usage"
-			dft:"dft-value"`
```
Comment (tuvia-akeyless) [inline tag_test.go:23]: need to keep this test with new lines

File: flag.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func (fl *flag) init(clr color.Color, dontSetValue bool) error {
 			}
 		}
 	}
-	if !dontSetValue && fl.tag.dft != "" && dft != "" {
+	if !dontSetValue && fl.tag.dft != "" && dft != "" && fl.tag.injectDft {
```
Comment (tuvia-akeyless) [inline flag.go]: It can be safer if the flag will be negative (`ignore_dft` etc...), and if true - ignore.

**Review by tuvia-akeyless — COMMENTED** [review]
Please add tests with inject="true" and inject=""
In addition, please check manually that it works correct via server mode & curl mode.

File: flag.go (hunk 2 above)
Comment (dan-akeyless) [reply flag.go]: It is better practice to use positive for tags

**Review by tuvia-akeyless — APPROVED** [review]

**Pull request merged**